docs/docusaurus/docs/core/run_validations/create_a_validation_definition.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Create a Validation Definition
+---
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+import GxData from '../_core_components/_data.jsx'
+import PrereqPythonInstalled from '../_core_components/prerequisites/_python_installation.md';
+import PrereqGxInstalled from '../_core_components/prerequisites/_gx_installation.md';
+import PrereqPreconfiguredDataContext from '../_core_components/prerequisites/_preconfigured_data_context.md';
+import PrereqPreconfiguredDataSourceAndAsset from '../_core_components/prerequisites/_data_source_and_asset_connected_to_data.md';
+import PrereqPreconfiguredExpectationSuiteAndExpectations from '../_core_components/prerequisites/_expectation_suite_with_expectations.md';
+
+import StepRequestADataContext from '../_core_components/common_steps/_request_a_data_context.md';
+
+
+A Validation Definition is a fixed reference that links a Batch of data to an Expectation Suite. It can be run by itself to validate the referenced data against the associated Expectations for testing or data exploration.  Multiple Validation Definitions can also be provided to a Checkpoint which, when run, executes Actions based on the Validation Results for each provided Validation Definition.
+
+## Prerequisites
+
+- <PrereqPythonInstalled/>.
+- <PrereqGxInstalled/>.
+- <PrereqPreconfiguredDataContext/>.
+- <PrereqPreconfiguredDataSourceAndAsset/>.
+- <PrereqPreconfiguredExpectationSuiteAndExpectations/>.
+
+<Tabs>
+
+<TabItem value="procedure" label="Procedure">
+
+1. Import the `ValidationDefinition` class from the {GxData.product_name} library:
+
+   ```python title="Python"
+   from great_expectations.core import ValidationDefinition
+   ```
+
+2. Request a Data Context:
+
+   ```python title="Python"
+   import great_expectations as gx
+   
+   context = gx.get_context()
+   ```
+
+3. Retrieve an Expectation Suite with Expectations.
+
+   Update the value of `suite_name` in the following code with the name of your Expectation Suite.  Then execute the code to retrieve that Expectation Suite:
+
+   ```python title="Python"
+   suite_name = "<NAME OF AN EXISTING EXPECTATION SUITE>"
+   suite = context.get_expectation_suite(suite_name)
+   ```
+
+4. Retrieve the Batch Definition that describes the data to associate with the Expectation Suite.
+
+   Update the values of `data_source_name`, `data_asset_name`, and `batch_definition_name` in the following code with the names of your previously defined Data Source, one of its Data Assets, and a Batch Definition for that Data Asset.  Then execute the code to retrieve the Batch Definition:
+
+   ```python title="Python"
+   data_source_name = "my_datasource"
```

Comment:
Maybe a philosophical thing that's already answered, but I think names that tell a bit more of a story might help the concepts stick better, e.g.
```
data_source_name = "my_postgres_datasource"
data_asset_name = "taxi_trips_table"
batch_definition_name = "taxi_trips_by_day"
```
Or something like that

Reply:
I don't think it's been directly answered, but my reasoning is: I try to leave the names as generic as possible to indicate that they can be substituted with anything the reader may have, instead.  In this case, the Data Source doesn't need to be a Postgres Data Source, or a File Data Source -- it can be any Data Source for any data format, and the request will work just the same.  Likewise for the Data Asset and the Batch Definition: it doesn't matter what the data in question is, it can be retrieved if you replace the generics with the names of your Data Asset and Batch Definition.

Reply:
Cool, that seems reasonable to me!